CONTRIBUTING.md

```diff
@@ -62,6 +62,16 @@ If you observe any error in tests with matching to above error message, that imp
 
 If you observe following error running any of the test files that means you are running a file which itself or any dependency of that file imports `vitest`, but you are not running that file with `vitest` runner. Try running it with `yarn vitest` command, not with `node` command.
 
+**Error: Could not locate the bindings file.**
+
+If you observe the above error in running tests or the [beacon chain cli](https://github.com/ChainSafe/lodestar/issues/1396#issue-677613531), try removing `node_modules` and running `npm rebuild`:
```

Comment:
doesn't `yarn rebuild` do the job as well? we should avoid using `npm`

Reply:
ah seems like this only works for a single module, we do this in our Dockerfile, but if you remove all `node_modules` why doesn't a simple `yarn install` do the job? or maybe `yarn install --force`